v1.1

andyb3 opened the pull request:
Update to address several issues as referenced in the commits.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/moka-guys/dnanexus_happy/16)
<!-- Reviewable:end -->

aledj2 commented on Readme.md:
```diff
@@ -31,18 +36,18 @@ Note:
 This app outputs:
 1. Summary csv file containing separate performance metrics for SNPs and Indels
 2. Detailed results folder containing:
```
does this folder contain more than that stated? raise #18

andyb3 replied:
Yes it does. I've updated the README to address this. See https://github.com/moka-guys/dnanexus_happy/pull/16/commits/7a0873ae5f39a93adf18d3b18a8835ecc467c5b6

aledj2 commented:
Reviewed 3 of 4 files at r1.
Review status: 3 of 4 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/code.sh, line 16 at r1](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16#-KyHOg4VjgLSzfNiRpNa:-KyHOg4WvifdJJ6UE29Q:b36ijkv) ([raw file](https://github.com/moka-guys/dnanexus_happy/blob/8fd7267057c1dcde2bfc4eaa1ac4b1a9b59dd142/src/code.sh#L16)):*
> ```Shell
> 
> #Add R to path
> export PATH='/home/dnanexus/R-3.4.2/bin:'$PATH
> ```

I assume this is bundled in the app? the folder name doesn't match that in the line `dx cat project-ByfFPz00jy1fk6PjpZ95F27J:file-F7k4g980777vqB04097vQkv0 | tar zxf - #R-3.4.2_ggplot2.tar.gz`

---

*[src/code.sh, line 52 at r1](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16#-KyHPRSmmi2GnfUWRCXf:-KyHPRSmmi2GnfUWRCXg:bpbkuzo) ([raw file](https://github.com/moka-guys/dnanexus_happy/blob/8fd7267057c1dcde2bfc4eaa1ac4b1a9b59dd142/src/code.sh#L52)):*
> ```Shell
> 
> #Zip and index VCFs
> bgzip $truth_vcf_path; tabix -p vcf ${truth_vcf_path}.gz
> ```

Any reason these are on seperate lines?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16)*
<!-- Sent from Reviewable.io -->

andyb3 commented:
Review status: 1 of 4 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/code.sh, line 16 at r1](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16#-KyHOg4VjgLSzfNiRpNa:-KyLHzIeoj9HG8tmwkQM:b-24eb0u) ([raw file](https://github.com/moka-guys/dnanexus_happy/blob/8fd7267057c1dcde2bfc4eaa1ac4b1a9b59dd142/src/code.sh#L16)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

I assume this is bundled in the app? the folder name doesn't match that in the line `dx cat project-ByfFPz00jy1fk6PjpZ95F27J:file-F7k4g980777vqB04097vQkv0 | tar zxf - #R-3.4.2_ggplot2.tar.gz`
</blockquote></details>

The archive name is R-3.4.2_ggplot2.tar.gz, but the extracted folder is R-3.4.2. I've updated the comments on each of the extractions showing the archive name and extracted file/folder name.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16)*
<!-- Sent from Reviewable.io -->

andyb3 commented:
Reviewed 1 of 2 files at r2.
Review status: 1 of 4 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/code.sh, line 52 at r1](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16#-KyHPRSmmi2GnfUWRCXf:-KyLuv25IIjTFAp-fCk1:b-ea8fgi) ([raw file](https://github.com/moka-guys/dnanexus_happy/blob/8fd7267057c1dcde2bfc4eaa1ac4b1a9b59dd142/src/code.sh#L52)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

Any reason these are on seperate lines?
</blockquote></details>

No reason for these being on separate lines. I've separated them.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16)*
<!-- Sent from Reviewable.io -->

andyb3 commented:
Review status: 1 of 4 files reviewed at latest revision, 4 unresolved discussions.

---

*[resources/home/dnanexus/rocplot_pdf.Rscript, line 6 at r3](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16#-KyWF2xGBMiPvR7gX5m4:-KyWF2xGBMiPvR7gX5m5:b-i0va4q) ([raw file](https://github.com/moka-guys/dnanexus_happy/blob/0f058465e8396db64ada380d2fce1ea8db621c50/resources/home/dnanexus/rocplot_pdf.Rscript#L6)):*
> ```
> # NOTE: The below script from Illumina has been modified to output as pdf rather than png in ggsave 
> # (because png output requires X11 windowing which isn't enabled in DNAnexus).
> ```

This script is a direct copy from the illumina/hap.py repository so shouldn't require code review: https://github.com/Illumina/hap.py/blob/d51d111e494b561b37c66299daf5a6c65a8d2ca9/src/R/rocplot.Rscript
The only modification I made is to output as a pdf rather than png for the reason stated in comments above.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16)*
<!-- Sent from Reviewable.io -->

aledj2 commented:
Reviewed 1 of 2 files at r2, 1 of 1 files at r3.
Review status: 3 of 4 files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16)*
<!-- Sent from Reviewable.io -->

aledj2 commented:
I haven't reviewed rocplotpdf_Rscript in full but I can see it is set to save as a pdf (last few lines of the file)

---

Reviewed 1 of 4 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/moka-guys/dnanexus_happy/16#-:-KyabFgZCuCzda2-0ucd:b-bc75uc)*
<!-- Sent from Reviewable.io -->
